css/style.css

```diff
@@ -0,0 +1,1465 @@
+  color: white;
+}
+
+.latest-deals-offer .offer-image:hover {
+  cursor: pointer;
+}
+
+.offer-image1 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/7a5a98dff30d5c38ecbc1700812eb2e1b93dce93/store/4b1f63289bd195a7bbafafbe36579fe597a5f1e7a0cb8165012908f3db74/sale_28088_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image2 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/1ce9b5e7afdcdd63a87d67d709a87ff735114cdd/store/3d847ed1b1c50758b3e478d2bbadfa527516bf72fa57658c5e8f7ffcc3f9/sale_162647_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image3 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp0.stackassets.com/c1fc5b0d89d9fb684e2ded4df2d6fbcbc3bf5bee/store/2a25d86229518c8634d2dae006e7aa58e735e292002cd2aabc0b17513a54/sale_24707_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image4 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/b34f2af9e9ec3122eb0c0b252532b61f8a08c0ea/store/243275a3bf8c7d20203d2e6e3b9f3664926f9f8913e755d1ae8811aecd9b/sale_25748_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image5 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/549384f77e88448bc914df09683a756f0c5e4c57/store/70bd980cd752af445b083eb46b1db2fb3fcee3c55eddbade2cae4193f5c5/sale_162254_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image6 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/131a74f7d56643b61ee30fb3e8da705c53031bed/store/b32bf58c9865347773f96c04f56c69f07227e61baeb48f8ffffec4861e5e/sale_18296_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image7 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/dd2a99b3f7d21cd700c3d210798ff74322c011d5/store/f4313846701bed0b9cbd6b0a983722fdce21faef621975d07c3a90a79e55/sale_148067_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
+
+.offer-image8 {
+  background-image: url(https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp1.stackassets.com/78d21e7906585957a1c4878636e0265d8bc3dffe/store/b39642357ca106ff7341d682ee27edcec23cbb3b7c2e9ddc690bb89533f1/sale_162717_primary_image.jpg);
+  object-fit: cover;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
There's a lot of repetition here. One way to remove the repetitions is by creating a single block for all the repeated CSS rules with a class name, then apply the class name to the element class list. i.e.
```CSS
.offer-image-default {
  object-fit: cover;
  background-size: cover;
  background-position: center;
}
```
Doing this will eliminate the duplication.